src/test/java/dev/blaauwendraad/masker/json/path/JsonPathParserTest.java

```diff
@@ -100,10 +100,7 @@ private static Stream<Set<String>> ambiguousJsonPaths() {
                 Set.of("$.a.b.c.f", "$.*.*.*.u"),
                 Set.of("$.*.b.c", "$.q.w.e", "$.*.d.f"),
                 Set.of("$.a.b.c", "$.d.*.*.f", "$.d.*.v.f"),
-                Set.of("$.a.b.*.*.d", "$.a.b.*.c"),
```

Comment:
This seems to be the one we should still not allow, no?

Reply:
Basically when you have the path `a.b.x.c` then it's still ambiguous

Reply:
That's "diff" issue, this pair is 4 lines below